src/risknet/run/model.py

```diff
@@ -124,7 +134,8 @@ def xgb_evaluate(eta, gamma, max_depth, min_child_weight, subsample, colsample_b
 
         self.bst = xgb.train(param, dtrain, num_boost_round=int(xgb_bo.max['params']['rounds']), evals =[(dval, 'val')],
                              maximize=maximize, verbose_eval=True) #.train returns a Booster object
-
+        
+        logger.info("CPU usage after training model: " + str(cpu_monitor.cpu_check()))
```

Comment:
why do we care about cpu usage pre/post training? I'd be more concerned with memory since there can be memory leaks but really, once the application is done running all memory gets released.